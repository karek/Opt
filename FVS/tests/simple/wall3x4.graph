a1 a2
a2 a3
a3 a1
b1 b2
b2 b3
b1 b3
c1 c2
c2 c3
c3 c1
d1 d2
d2 d3
d3 d1
a1 b1
c1 d1
a3 b3
c3 d3
a2 d2
b2 c2

