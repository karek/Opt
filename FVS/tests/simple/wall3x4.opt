4
a1 b2 c1 d2 
