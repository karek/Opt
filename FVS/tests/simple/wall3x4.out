4
a3 b3 c3 d2 
